src/response/file.rs

```diff
@@ -25,7 +25,7 @@ const BASE_DELAY_IN_MILLIS: u64 = 100;
 
 /// The amount of times that we'll retry to open a file if in an error occurs.
 ///
-const MAX_ATTEMPTS: u64 = 3;
+const MAX_ATTEMPTS: u64 = 5;
```

Comment:
Heads up, I'm slightly biased against not wanting to make the fastest file server implementation in favor of an API that is fun to use, correct, more than fast enough, easy to audit, and easy to contribute to.

I decided to increase attempts here as it takes slightly longer for `FileStream` to get dropped than it did to explicitly set `this.file = None`. We could of course, work with a raw file descriptor but that could become overly complicated.